app/src/util.ts

```diff
@@ -19,3 +19,19 @@ export function unionAuthors(blob: HydratedGitBlobObject) {
 export function classes(...args: (string | undefined)[]) {
   return args.filter((x) => x).join(" ")
 }
+
+export function dateFormat(epochTime: number | undefined) {
+  if (!epochTime) return "Invalid date"
+  return new Date(epochTime * 1000).toLocaleString("en-gb", {
```

Comment:
Consider using `[navigator.languate](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language)`, instead of hardcoded locale.